### PR TITLE
kodiPackages.mediathekview: init at 1.0.9

### PR DESCRIPTION
**Pull request by dschrempf**
## Description of changes
Add [MediathekView (for Kodi)](https://mediathekview.de/download/), which allows
access to media libraries of German speaking broadcasting stations.

I tested this on my home server. However, I do not know if the automatic updates
work (never tried this functionality).

I am looking forward to your comments!

@NixOS/kodi

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by aanderse** [comment]
thanks for contributing!

... and sorry for the delay

File: pkgs/applications/video/kodi/addons/mediathekview/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    repo = namespace;
+    rev = "release-${version}";
+    hash = "sha256-XYyocXFTiYO7Ar0TtxjpCAy2Ywtnwb8BTxdKxwDWm4Y=";
+  };
+
+  propagatedBuildInputs = [
+    myconnpy
+  ];
+
+  passthru.updateScript = addonUpdateScript {
```
Comment (aanderse) [inline pkgs/applications/video/kodi/addons/mediathekview/default.nix:19]: this slipped by me... your other PR reminded me this isn't necessary

see https://github.com/NixOS/nixpkgs/pull/291065 as an example - `fetchFromGitHub` is enough to make the bot update the package

if you don't mind can you please remove this whenever you get a chance - it serves no purpose and can lead people down the wrong path

thank you and sorry for missing! 🙇‍♂️

Comment (dschrempf) [reply pkgs/applications/video/kodi/addons/mediathekview/default.nix:19]: Good to know, thank you! I will change that.

Comment (dschrempf) [reply pkgs/applications/video/kodi/addons/mediathekview/default.nix:19]: Btw, there are quite a lot of addons with this directive. Should we remove them too?

Comment (aanderse) [reply pkgs/applications/video/kodi/addons/mediathekview/default.nix:19]: that sounds like a mistake yeah

if a package was originally using the kodi repo for source and then switched to github i can see how that would get missed pretty easily

**Pull request merged**